src/libertem/api.py

```diff
@@ -195,6 +195,10 @@ def create_mask_analysis(self, factories, dataset, use_sparse=None,
         dtype : numpy.dtype, optional
             Specify the dtype to do the calculation in. Integer dtypes are possible if
             the numpy casting rules allow this for source and mask data.
+        _use_udf : bool, optional
+            This temporary inofficial parameter allows to switch between the Job and the UDF
+            implementations for testing and comparison purposes until the Job API is officially
+            deprecated and the UDF implementation is used exclusively. See :issue:`549`
```

Comment:
Best to not have this parameter in the API, as we only need it for testing and maybe performance comparison. Same can be done by manually creating and tweaking the `Analysis` instance.

Reply:
Good point, thx! :-)